test/models/articleSpec.js

```diff
@@ -30,18 +30,18 @@ describe('Article model', function(){
             var article = new models.Article(fixtures.article);
             expect(article.body).not.to.contain('href="http://"');
         });
-    
+
         // This is specifically for Next. We don't support all types of content from day 1.
         it('Remove links that are not Content API articles from the body', function() {
             var article = new models.Article(fixtures.article);
             var $ = cheerio.load(article.body);
             expect($('a').length).to.equal(2);
         });
-    
+
         it('Calculate the article word count and estimated reading time', function() {
             var article = new models.Article(fixtures.article);
-            expect(article.wordCount).to.equal(775);
-            expect(article.readingTime).to.equal(3); // in minutes
+            expect(article.wordCount).to.equal(741);
+            expect(article.readingTime).to.equal(2); // in minutes
         });
```

Comment:
does that mean that one of the links in the fixture article has been 'undefined' this whole time?